### DISCUSSION TITLE
Month Picker - Add-on/customization of date picker or new component

### DISCUSSION THREAD

**Discussion by rebeccaalpert** [patternfly/patternfly#5573] [Feature requests]
**Is this a new component or an extension of an existing one?**
Right now, PatternFly offers a [date picker](https://www.patternfly.org/v4/components/date-picker/), but not a month picker. We have a workaround for now, but we were wondering if PatternFly would be interested in a month picker contribution. We needed one for our app and wound up forking/customizing the PatternFly date picker to do what we needed.

**Describe the feature**
PatternFly has a date picker, but not a month picker. We needed to allow users to filter data by a month rather than a specific date. My team was using [react-month-picker](https://github.com/nickeljew/react-month-picker) before, but ran into some issues in the context of PatternFly layout components. It also couldn't be used with a keyboard. 

We pulled in the date picker code from PatternFly and modified it to do what we needed for now; here is a screenshot from our app:

<img width="343" alt="Screen Shot 2023-04-21 at 11 29 00 AM" src="https://user-images.githubusercontent.com/7014965/233676046-02a0f8d2-ee5b-4436-a9c7-69777f5eb38b.png">

I wanted to open a conversation here to find out if you would be interested in an upstream contribution, and if so, what would be involved.

**Comment by mcarrano** [comment]
Converting for more discussion
